### DISCUSSION TITLE
Does this project almost will be dead

### DISCUSSION THREAD

**Discussion by khsh90** [jonataslaw/getx#2809] [Q&A]
Does this project almost will be dead because no recent update available the last one beautiful 12-month.

Waiting for your appreciated reply
Regards

**Comment by dasaintgray** [comment]
Sir @jonataslaw, we are waiting for your reply, are you giving up your project?

**Comment by jonataslaw** [comment]
There is updates, but in release candidate version of pub.dev, and in github. 
I am collecting feedbacks in the RC to send it to stable.

**Reply by crazidev** [reply]
Please can you split the Getx into different parts,Getx state manage, Getx route, Getx dependency injection, and Getx utilities, this would be better because we don’t actually use everything in the package

**Reply by afzalali15** [reply]
We are eagerly waiting for update. 
Our organization is loosing trust as the package has not received any update since more than a year now.

We are afraid that this will soon become inconsistent with Flutter stable channel and we will have to re write whole app again. Please plan regular updates ! Appreciate your time and effort on this awesome package.

**Comment by davidbalu10** [comment]
What date do you plan to release version 5?

**Comment by jonataslaw** [comment]
I am sorry for the delay, I am working on next update right now.

**Reply by afzalali15** [reply]
That's exciting!
I will be among first few to test out the latest release 😃

**Reply by nugrahaz** [reply]
i feel getx in flutter 3.13 feels unstable, i think some code is deprecated

**Comment by iSaqibShafique** [comment]
> I am sorry for the delay, I am working on next update right now.

Thank you so much sir, I don't know how can I explain the pain when someone say GetX is going to dead. 😢
